pandas/_libs/reduction.pyx

```diff
@@ -529,13 +531,14 @@ def apply_frame_axis0(object frame, object f, object names,
             item_cache.clear()  # ugh
 
             object.__setattr__(slider.dummy, 'name', names[i])
-            piece = f(slider.dummy)
+            if i >= int(skip_first_index):
```

Comment:
I would add a comment (here or above) about what you are doing, and a reference to the issue.